applications/nrf5340_audio/src/bluetooth/ble_audio_services.c

```diff
@@ -454,9 +463,17 @@ int ble_vcs_discover(struct bt_conn *conn, uint8_t channel_num)
 }
 #endif /* (CONFIG_BT_VCP_VOL_CTLR) */
 
+void ble_mcs_discover_called_set(bool discovery_called)
+{
+#if (CONFIG_BT_MCC)
+	mcs_discover_called = discovery_called;
+#endif
+}
+
 int ble_mcs_discover(struct bt_conn *conn)
 {
 #if (CONFIG_BT_MCC)
+	mcs_discover_called = true;
```

Comment:
Consider using `ble_mcs_discover_called_set` for consistency 
```suggestion
	ble_mcs_discover_called_set(true);
```

Reply:
Removed the function, so just setting it locally in this file now